src/authorized_keys/views.py

```diff
@@ -32,6 +32,17 @@ class ReverseServerAuthorizedKeysViewSet(viewsets.ModelViewSet):
     serializer_class = ReverseServerAuthorizedKeysSerializer
     permission_classes = [IsAuthenticated]
 
+    def perform_create(self, serializer):
```

Comment:
The `perform_create` seems a little bit redundant, is it able to extract the method to a super class, then make all the `ViewSet` inherit it?

Reply:
Add parent class for inheriting in this commit https://github.com/NatLee/telepy/pull/17/commits/fb29e2e06ae68b70d5befdd3f3545085ad310c4e :D